objectivec/include/ort_session.h

```diff
@@ -64,6 +64,22 @@ NS_ASSUME_NONNULL_BEGIN
                                                    runOptions:(nullable ORTRunOptions*)runOptions
                                                         error:(NSError**)error;
 
+/**
+ * Gets the model's input names.
+ *
+ * @param[out] error Optional error information set if an error occurs.
+ * @return An array of input names, or nil if an error occurs.
+ */
+- (nullable NSArray<NSString*>*)inputNamesWithError:(NSError**)error;
+
```

Comment:
nit: Can we add support for getting the overrideable initializers names as well for completeness (i.e. equivalent to the C++ API's Session::GetOverridableInitializerName)? 

They're also valid inputs so potentially a significant gap to not have way to fetch those.